src/ui/web/displayOnFocus/displayOnFocus.stories.tsx

```diff
@@ -44,10 +47,12 @@ const Template: ComponentStory<FunctionComponent> = () => (
   </Fragment>
 )
 
-export const Default = Template.bind({})
-Default.storyName = 'displayOnFocus'
-Default.play = async () => {
-  await screen.findByRole('button') // wait first render
+export const Default: Story = {
+  render: () => <StoryComponent />,
```

Comment:
étant donné que la fonction `StoryComponent` retourne un élément JSX, est-il nécessaire de rewrapper la fonction ?

pourrait-on faire ceci ?
```suggestion
  render: StoryComponent,
```

Reply:
pareil pour d'autres fichiers

Reply:
@xlecunff-pass on a fait un ticket pour ça hier

Reply:
on a pas mal de cas comme ça: `render: () => <SectionWithSwitch title="Section with switch title" toggleLabel="Toggle label" />`

Si l'on souhaite ajouter des props, il faudra réécrire sous forme de fonction
Dans un soucis d'uniformité je préfère laisser tel quel pour le moment, on pourra en rediscuter

Reply:
la dernière fois que j'avais relu la PR, on avait des `render: <UnComposant />` et aussi des `render: UnComposant`, ce n'était pas uniforme